robottelo/api/utils.py

```diff
@@ -825,6 +826,25 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             self.temp.update(['locked'])
 
 
+@contextmanager
+def satelliteSetting(key_val: str):
```

Comment:
why camel case ? you `update_setting` / `satelliet_setting` ?

Reply:
moved to `satellite_setting`.